docs/ELEVATION.md

```
@@ -0,0 +1,60 @@
+- China: ~600MB (15M points)
+- India: ~300MB (8M points)
+
+## Example
+
+```bash
+# Download Japan elevation grid
+python scripts/download_elevation_grid.py --country JP
+
+# Download Korea with 2km resolution
+python scripts/download_elevation_grid.py --country KR --resolution 0.02
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)